osf-packager/src/test/java/org/dataconservancy/cos/packaging/OsfContentProviderTest.java

```diff
@@ -65,19 +59,12 @@ protected String getOsfServiceConfigurationResource() {
 
     @Test
     public void testCreatePackageSimple() throws Exception {
-        final OsfPackageGraph packageGraph = new OsfPackageGraph(ontologyManager);
-        factory.interceptors().add(new RecursiveInterceptor(testName, IpmPackagerTest.class, getBaseUri()));
+        // Prepare the OSF registration and users information
```

Comment:
Both of these unit tests pass, in that they generate package files that are not mechanically inspected.  But the contents of those files do not appear to be complete, as was the case (I believe) when I last worked on this code to add the Wiki content.  More work may be needed (probably below the level of these changes) to ensure that the packages are complete.  The forthcoming RdfContentProviderTest _does_ successfully populate the packages that it creates with the same IpmPackager that is used here.